src/components/Persona/Persona.scss

```diff
@@ -33,14 +46,14 @@ $ms-color-initials-darkRed:     #b91d47;
 }
 
 .ms-Persona-imageArea {
-  position: relative;
-  display: block;
-  overflow: hidden;
-  text-align: center;
-  width: 48px;
-  height: 48px;
-  border-radius: 50%;
-  z-index: $ms-zIndex-back;
+	position: relative;
```

Comment:
Same feedback as above--love the indenting, but it should be 2 spaces instead of a 4-space tab.